test/CMakeLists.txt

```diff
@@ -28,6 +22,17 @@ if(LIBCORO_FEATURE_NETWORKING)
     )
 endif()
 
+if(LIBCORO_FEATURE_THREADING)
+    list(APPEND LIBCORO_TEST_SOURCE_FILES
+        bench.cpp
+        test_semaphore.cpp
```

Comment:
I see there is an old usage of `coro::io_scheduler` in here for using `yield()` but I added `yield()` onto `coro::thread_pool` recently, how would you feel about updating this test so this file could be moved outside of the `LIBCORO_FEATURE_THREADING` flag?

Reply:
Nice! I'm always in favor of including more tests. Done.